### PR TITLE
Add FORM_ID to .env.example

### PR DESCRIPTION
**Pull request by Adhiraj10**
Since all of the new contributors were facing an issue(including me) where they could not run this project locally because of the FORM_ID environment variable not being serialized, I modified it to a random number so that the project can run locally. This is not a permanent fix but it should help new contributors with getting started.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const getStaticProps = async () => {
   const date = new Date();
   const selectedImage = date.getDay();
   const image = heroImages[selectedImage];
-  const FORM_ID = process.env.FORM_ID;
```
Comment (akshitarora921) [inline pages/index.js:20]: I think the best way to fix it right now is to add FORM_ID in `.env.example` with random string

**Review by akshitarora921 — CHANGES_REQUESTED** [review]
@Adhiraj10 Thank you for raising the pull request.

File: pages/index.js (hunk 1 above)
Comment (Adhiraj10) [reply pages/index.js]: @akshitarora921 I have made the appropriate changes.
@LowLifeArcade Happy to help :)

**Comment by Adhiraj10** [comment]
I have made modifications to the code as suggested by @akshitarora921

**Comment by Adhiraj10** [comment]
In commit #0c8a392, I had also added dotenv package as a dependency. But @akshitarora921 made me realize that we don't really need dotenv to run the project locally and by simply initializing the FORM_ID variable to a dummy value in .env.example is enough. Therefore the next commit #a7e65cd

**Comment by LowLifeArcade** [comment]
Awesome @Adhiraj10. Thank you 🙏 
Merging.

**Pull request merged**